apps/web/src/scenes/UserGalleryPage/UserNameAndDescriptionHeader.tsx

```diff
@@ -87,8 +94,12 @@ export function UserNameAndDescriptionHeader({ userRef, queryRef }: Props) {
   const userBadges = useMemo(() => {
     if (!badges) return [];
 
+    if (!isActivityBadgeEnabled) {
+      return badges.filter((badge) => badge && badge?.contract && badge?.imageURL);
+    }
```

Comment:
nit: should just be able to do `badge?.contract && badge?.imageURL`, and drop `badge &&`

Reply:
You're right! removed it